gulpfile.js

```diff
@@ -28,47 +28,44 @@ var languages = {
   js: {
     repo: 'Asana/node-asana',
     branch: 'api-meta-incoming',
-    templatePath: 'src/templates',
-    outputPath: 'lib/resources/gen'
+    outputPath: 'lib/resources/gen',
+    preserveRepoFiles: false
   },
   php: {
     repo: 'Asana/php-asana',
     branch: 'api-meta-incoming',
-    templatePath: 'templates',
-    outputPath: 'src/Asana/Resources/Gen'
+    outputPath: 'src/Asana/Resources/Gen',
```

Comment:
It should match whatever's in the repo, and in PHP-land the convention is upper case.